src/metabase/query_processor/middleware/cache.clj

```diff
@@ -100,7 +100,7 @@
        (a/put! in-chan (if (map? result) (m/dissoc-in result [:data :rows]) {}))
        (a/close! in-chan)
        (let [duration-ms (- (System/currentTimeMillis) start-time)]
-         (log/info (trs "Query took {0} to run; miminum for cache eligibility is {1}"
+         (log/info (trs "Query took {0} to run; minimum for cache eligibility is {1}"
```

Comment:
Hawk's eye!
I was checking this on my phone where the word isn't highlighted. Took me good 2 minutes to figure it out 🙃